cross/p7zip/PLIST

```diff
@@ -1,9 +1 @@
-rsc:bin/7z
-rsc:bin/7za
-rsc:bin/7zr
-lib:lib/p7zip/7z
-lib:lib/p7zip/7za
-lib:lib/p7zip/7zCon.sfx
-lib:lib/p7zip/7zr
-lib:lib/p7zip/7z.so
-lib:lib/p7zip/Codecs/Rar.so
+bin:lib/p7zip/7za
```

Comment:
Two questions:
1. The path looks odd?  it's a `bin` in a `lib` folder?  And guessing staticaly compiled to be autonomous?
2. No other packages requires it?

Reply:
1. Yeah strange indeed, it's just the output of the p7zip build for many years so I didn't look at it or update it.
2. When searching for `cross/p7zip` in the repo, it's only used by SABnzbd.

Reply:
This would not have worked, as `rsc:/bin/7za` was a link to `bin:lib/p7zip/7za`.

I have reworked `cross/p7zip` to build 7za (the default target) only.